.github/workflows/build-m1-binaries.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           if [[ ${GITHUB_REF_NAME} = *-rc[0-9]* ]]; then
             echo "CHANNEL=test" >> "$GITHUB_ENV"
           fi
+      - name: Set Release CHANNEL (for release)
+        if: ${{ github.event_name == 'pull_request' && startsWith(github.base_ref, 'release') }}
```

Comment:
Hmm, so we want to run it on PR, but not on the pushes to the branch? Why?

Reply:
Thank you! Done